client/src/panels/panel_declarations/services/provided_services_list.js

```diff
@@ -1,30 +1,80 @@
-import text from "./services.yaml";
+import text from "./provided_services_list.yaml";
 import { Service } from "../../../models/services.js";
+import { create_text_maker } from "../../../models/text.js";
+import { TM } from "../../../components/index.js";
 
-import {
-  create_text_maker_component,
-  InfographicPanel,
-  declare_panel,
-} from "../shared.js";
+import { InfographicPanel, declare_panel } from "../shared.js";
+import { FancyUL } from "../../../components";
 
-const { text_maker, TM } = create_text_maker_component(text);
+const text_maker = create_text_maker(text);
```

Comment:
any reasons why you needed to import `TM` separately?

Reply:
Whoops. I guess I had a different idea of what the method did earlier haha